units/systemd-timedated.service.in

```diff
@@ -14,6 +14,7 @@ Documentation=man:localtime(5)
 Documentation=man:org.freedesktop.timedate1(5)
 
 [Service]
+Type=notify
```

Comment:
note that this defaults to Type=dbus if BusName= is specified, like it is here. Which is also an OK way to do ready notification.

That said, I figure sooner or later we want to have this stuff available via varlink too, and thus accessible during early boot, and then Type=dbus won't work anymore. Hence it sounds good to switch this over already.